src/bindings/python/docs/contributing.md

```diff
@@ -1,30 +1,29 @@
 # Contributing to OpenVINO™ Python API
 
 #### Prerequisites
-*To be added...*
 
 ##### Enviroment
 In case the Python version you have is not supported by OpenVINO, you can refer to [openvino/src/bindings/python/docs/python_version_upgrade.md](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/docs/python_version_upgrade.md) for instructions on how to download and build a newer, supported Python version.
-<!-- TODO: Link to enviroment setup -->
-*To be added...*
+
+The environment setup is described in [the example on the build.md page](./build.md#Example:_using_pyenv_with_OpenVINO™_on_Linux_based_system).
 
 ##### Building
-Building instructions can be found in [BUILDING.md](../BUILDING.md)
+Building instructions can be found in [build.md](./build.md#_Building_the_OpenVINO™_Python_API).
 
 ## Contribution guidelines and best practices
 
 #### How to contribute to Python API?
 It is nothing special... :) First, make sure that all prerequisites are met and focus on writing the code itself. A good starting point is to have some knowledge of the Python language. C++ is also a vital language for OpenVINO™, so it is not a surprise that it is used in this part of the project as well.
 
 Code snippets and detailed explanations can be found here:
-<!-- Link to EXAMPLES -->
-    openvino/src/bindings/python/docs/code_example.md
+
+[Examples of OpenVINO™ Python API code](./code_example.md)
 
 ##### Always test out our code! Don't forget about it before pushing and triggering CIs.
 
 Please refer to Test Guide available here:
```

Comment:
To learn how to test your code, refer to the guide on
[how to test OpenVINO™ Python API](./test_examples.md#Running_OpenVINO™_Python_API_tests)

Reply:
done